src/components/create-card.jsx

```diff
@@ -1,9 +1,76 @@
 import React from 'react';
 
 class CreateCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      question: '',
+      answer: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+  }
+
+  handleReset() {
+    this.setState({
+      question: '',
+      answer: ''
+    });
+    this.props.setView('view-cards');
+  }
+
+  handleChange(event) {
+    if (event.target.name === 'question') {
+      this.setState({
+        question: event.target.value
+      });
+    } else {
+      this.setState({
+        answer: event.target.value
+      });
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const newCard = {
+      question: this.state.question,
+      answer: this.state.answer
+    };
+    this.props.addCard(newCard);
+    this.setState({
+      question: '',
+      answer: ''
+    });
```

Comment:
YOu don't need to clear state since you are calling `handleReset` which already resets the state....